src/mono/wasm/debugger/tests/debugger-test/debugger-array-test.cs

```diff
@@ -345,4 +345,51 @@ public static void run()
             Console.WriteLine($"int_arr: {int_arr_3.Length}");
         }
     }
+
+    public class InlineArray
+    {
+        class One {}
+        class Two {}
+        class Three {}
+        class Four {}
+        struct E
+        {
+            public int x;
+            public int y;
+            public object o;
+        }
+
+        [System.Runtime.CompilerServices.InlineArray(Length)]
+        struct Arr1
+        {
+            public const int Length = 42;
```

Comment:
Add a static field here too, and check that in the test. This would ensure that the inline array access is not disturbed by the static fields, as we check for that also when looking up the inline array values.